static/build/.tmp_update/script/scraper/menu.sh

```diff
@@ -162,40 +226,36 @@ Menu_Config_ScrapingSelection()
     
     
     # Recovering the values of variables
-    Retroarch_enabled=$(echo "$json" | jq -r '.Retroarch_enabled')
     Screenscraper_enabled=$(echo "$json" | jq -r '.Screenscraper_enabled')
     Launchbox_enabled=$(echo "$json" | jq -r '.Launchbox_enabled')
-    
+    Retroarch_enabled=$(echo "$json" | jq -r '.Retroarch_enabled')
```

Comment:
The initial order has a logic : first the fastest scrapers. 
imo screenscraper should be the lastest in this order now that launchbox script is fast.
Your idea to let the user choose the order will be useful.